Source/Adobe.Target.Client/Util/CollectionUtils.cs

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+namespace Adobe.Target.Client.Util
+{
+    using System.Collections.Generic;
+
+    internal static class CollectionUtils
```

Comment:
Things like this should be put in an Extension Method class, check https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/how-to-implement-and-call-a-custom-extension-method

Reply:
Good catch, fixed